packages/server/start/mpeSocket.ts

```diff
@@ -1,9 +1,33 @@
-import { MpeRoomClientToServerCreateArgs } from '@musicroom/types';
+import {
+    MpeRoomClientToServerAddTracksArgs,
+    MpeRoomClientToServerCreateArgs,
+} from '@musicroom/types';
 import MpeRoomsWsController from 'App/Controllers/Ws/MpeRoomsWsController';
+import MpeRoom from 'App/Models/MpeRoom';
 import SocketLifecycle from 'App/Services/SocketLifecycle';
 import Ws from 'App/Services/Ws';
 import { TypedSocket } from './socket';
 
+interface IsUserInMpeRoomArgs {
+    userID: string;
+    roomID: string;
+}
+
+async function isUserInMpeRoom({
```

Comment:
Should put this logic inside the `SocketLifeCycle.getSocketConnectionCredentials` method ?
We could add an optionnal parameter to the function such as `relatedMpeRoomID`, via a custom lucid query we could return the matching mpe room or compute and return a boolean `userIsInGivenMpeRoom` ?